docs/sysadmin/portlet-admin/Portlet-setup.md

```diff
@@ -0,0 +1,12 @@
+# Portlet Setup
+After a portlet-definition has been imported into uPortal, users in the Portal Adminstrators group can incorporate portlets into the user experience.
+
+To access, click to the _Portlet Administration Launcher_.  Select a portlet and click _Edit_.
```

Comment:
Okay...

Traditionally, uPortal has not provided much (if any) of this sort of documentation.  This is documentation about how to use the UI, and it's a bit of a can of worms:

  - It is brittle;  it's likely to become incorrect as the platform evolves
  - It is a truly massive undertaking, if we start to hold ourselves to the standard of providing it;  we probably don't have the bandwidth to tackle that

We normally aim to make the UI simple & intuitive enough that we don't need this kind of documentation.  (The question of whether we succeed at that is another matter.)  Typically, I would be happier to see investment in making the UI easier to "figure out" than in documentation about what to click.

Reply:
Ok, I'll back off that change.